src/asInput/index.jsx

```diff
@@ -50,8 +53,17 @@ const asInput = (WrappedComponent, labelFirst = true) => {
 
       if (!this.state.isValid) {
         desc.error = (
-          <div className={styles['form-control-feedback']} id={errorId} key="0">
-            {this.state.validationMessage}
+
+          <div className={classNames(styles['form-control-feedback'], { [styles['invalid-feedback']]: this.props.theme.includes('danger') })} id={errorId} key="0">
```

Comment:
I can pull out the `this.props.theme.includes('danger')` condition into a variable, as the condition is being used twice here, but I was interested to see whether it would be an issue as we add more themes. Is the idea that, as we add more themes, we conditionally apply classnames this way? I'll add a 'hasDangerTheme' variable if this looks good to everyone, otherwise.